code/game/atoms_movable.dm

```diff
@@ -241,6 +250,35 @@
 	// Update on_moved listeners.
 	INVOKE_EVENT(on_moved,list("loc"=NewLoc))
 
+/atom/movable/proc/animate_glide(Dir, dist)
+	if(!animate_movement)
+		return
+
+	var/const/FUDGE_ANIM = 0.1 //For unknown reasons, movement is all janky unless the animation is set to be slightly longer than it should have to be. This value is added to the animation duration.
+	#define GLIDE_HELPER(O, axis, val) O.axis -= val; animate(O, axis = val, world.tick_lag * WORLD_ICON_SIZE / (glide_size || 4) + FUDGE_ANIM, flags = ANIMATION_PARALLEL | ANIMATION_RELATIVE)
```

Comment:
Any reason why you are defining this locally ?

Reply:
Because it's not useful anywhere else and defines (marginally) slow down compilation based on how much code they span

Reply:
Because it's only used within `animate_glide`?